lib/ui/platform_dispatcher.dart

```diff
@@ -1290,7 +1291,11 @@ class PlatformConfiguration {
 class ViewConfiguration {
   /// A const constructor for an immutable [ViewConfiguration].
```

Comment:
Explain in the doc that only one of `view` and `window` should be non-null, and that they're equivalent.

Reply:
You can resolve this if you're satisfied with the doc added in 278f7249916621f6611e8da715e76511eae39178.